src/cdk-experimental/menu/menu-item-trigger.ts

```diff
@@ -86,20 +86,25 @@ export class CdkMenuItemTrigger implements OnDestroy {
     return this._overlayRef ? this._overlayRef.hasAttached() : false;
   }
 
+  /** Get a reference to the rendered Menu if the Menu is open and it is visible in the DOM. */
+  getRenderedMenu() {
```

Comment:
I would probably just make this `getMenu` and explain in the JsDoc that it returns `null` when the menu is closed